### PR TITLE
Init animejs

### PR DESCRIPTION
**Pull request by nzcodarnoc**
Install animation library and do a simple proof of concept.

### CODE REVIEW

**Review by JacOng17 — COMMENTED** [review]
Not too sure how the page is supposed to appear, but I get this: 
image
It spans more than my page & can only see the full title when I scroll far right:
image

File: pages/index.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
+import { useEffect } from "react";
+import Head from "next/head";
+import anime from "animejs";
+
+const Home = () => {
+  useEffect(() => {
+    anime({
+      targets: "div",
+      translateX: 250,
```
Comment (JacOng17) [inline pages/index.js:9]:
```suggestion
```

Comment (JacOng17) [reply pages/index.js:9]: Turns out better if that's what you actually wanted?
image

Comment (nzcodarnoc) [reply pages/index.js:9]: Oh yeah, it's a dumpster fire! 

All that I wanted to prove was that it animated. 

The final frame is a hot mess, but seeing as it's just a POC and going to be deleted, it didn't seem worth expending effort on it.

Comment (nzcodarnoc) [reply pages/index.js:9]: So: if you see that animation, that's all this PR was about. Not the final layout.

**Review by JacOng17 — APPROVED** [review]
LGTM, animation works! :tada:

**Pull request merged**